certbot/certbot-dns-renew.sh

```diff
@@ -1,25 +1,65 @@
 #!/usr/bin/env sh
 
 ACTUAL_DOMAIN="_acme-challenge.$CERTBOT_DOMAIN"
-SUBDOMAIN=$(echo -n "$ACTUAL_DOMAIN" | sed 's/.aleemhaji.com//')
+SUBDOMAIN="$(printf "%s" "$ACTUAL_DOMAIN" | sed 's/.aleemhaji.com//')"
```

Comment:
Well this file got a lot more complicated, but I guess _technically_ might be a bit more debuggable/readable?